Fix Typos (#34297)
demisto/content: Packs/AbnormalSecurity/ReleaseNotes/2_2_8.md

#### Integrations

##### Abnormal Security

> - Fixed an issue with the **abnormal-security-submit_false_positive_report** command by removing an extra semicolon in the *portal_link* argument

Reviewer: ```suggestion
- Fixed an issue in the **abnormal-security-submit_false_positive_report** command where the *portal_link* argument did not work as expected.
```

> - Fixed an issue with the **abnormal-security-submit_false_negative_report** command by removing an extra semicolon in the *recipient_email* argument

Reviewer: ```suggestion
- Fixed an issue in the **abnormal-security-submit_false_negative_report** command where the *recipient_email* argument did not work as expected.
```
